src/main/java/com/plaid/client/response/AssetReportGetResponse.java

```diff
@@ -0,0 +1,181 @@
+package com.plaid.client.response;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * See https://plaid.com/docs/api/#assets for more detail into the returned schema
+ **/
+public class AssetReportGetResponse extends BaseResponse {
+  private Report report;
+  private List<Warning> warnings;
+
+  public Report getReport() { return this.report; }
+  public List<Warning> getWarnings() { return this.warnings; }
+
+  public final static class Report {
```

Comment:
I was looking at the response body of `/asset_report/get`:

![image](https://user-images.githubusercontent.com/6411453/41778794-d299629c-75fd-11e8-91a5-b0cf07e87ef3.png)

This will deserialize as `report` and not `asset_report` right? I can name the field `assetReport` and use an annotation to deserialize it as `report`. What do you think?

Reply:
the class name doens't matter right?